tests/dist/chats.spec.dev.js

```diff
@@ -136,38 +141,40 @@ describe('chat resolvers', function () {
   test('send message to direct chat', function _callee3() {
     var sendMessageToDirectChatResponse, sendMessageToADirectChatData;
     return regeneratorRuntime.async(function _callee3$(_context3) {
-      while (1) {
+      const condition = 1;
+      while (condition) {
         switch ((_context3.prev = _context3.next)) {
           case 0:
             _context3.next = 2;
             return regeneratorRuntime.awrap(
-            axios.post(
-              URL,
-              {
-                query: '\n        mutation{\n          sendMessageToDirectChat(chatId: "'.concat(
-                  createdDirectChatId,
-                  '", messageContent: "this is a test message"){\n            _id\n          }\n        }\n            '
-                ),
-              },
-              {
-                headers: {
-                  Authorization: 'Bearer '.concat(token),
+              axios.post(
+                URL,
+                {
+                  query: '\n        mutation{\n          sendMessageToDirectChat(chatId: "'.concat(
+                    createdDirectChatId,
+                    '", messageContent: "this is a test message"){\n            _id\n          }\n        }\n            '
+                  ),
```

Comment:
It would be ideal to use a template literal here to improve the readability of this query.